script.js

```diff
@@ -92,7 +94,91 @@ function printCards() {
 					</div>
 				</div>`;
 		featuredSpeaker.innerHTML += markup;
+		if (initialLoad === data.length - 4) {
+			loadMoreBtn.classList.add('hidden');
+		}
 	}
+	initialLoad += 4;
 }
-
 printCards();
+
+loadMore.addEventListener('click', printCards);
+
+// Function to load timer in hero section
+
+function countDown() {
+	const now = new Date();
+	const eventDate = new Date(2022, 1, 24, 9);
+	const currentTime = now.getTime();
+	const eventTime = eventDate.getTime();
+
+	const remTime = eventTime - currentTime;
+	let s = Math.floor(remTime / 1000);
+	let m = Math.floor(s / 60);
+	let h = Math.floor(m / 60);
+	const d = Math.floor(h / 24);
+
+	h %= 24;
+	m %= 60;
+	s %= 60;
+
+	h = h < 10 ? `0${h}` : h;
+	m = m < 10 ? `0${m}` : m;
+	s = s < 10 ? `0${s}` : s;
+	document.getElementById('days').textContent = d;
+	document.getElementById('hours').textContent = h;
+	document.getElementById('minutes').textContent = m;
+	document.getElementById('seconds').textContent = s;
+
+	setTimeout(countDown, 1000);
+}
```

Comment:
Great job adding extra feature to the project:+1:
-  [ ] Let's stick to JavaScript best practices by maintaining two-space indentation.

Reply:
- [ ] While your video recording is so great, make sure to have a video of yourself showing at the bottom of the screen. Also, highlight a piece of code that you are very proud of in the video. This can easily tempt a potential recruiter to consider hiring you.
Happy coding:tada: